seahorse-workflow-executor/deeplang/src/main/scala/ai/deepsense/deeplang/DOperation.scala

```diff
@@ -98,9 +102,37 @@ abstract class DOperation extends Operation
   def inferGraphKnowledgeForInnerWorkflow(context: InferContext): GraphKnowledge = GraphKnowledge()
 
   def typeTag[T : ru.TypeTag]: ru.TypeTag[T] = ru.typeTag[T]
+
+  val reportType: ChoiceParam[ReportType] = ChoiceParam[ReportType](
+    name = "report type",
+    description = Some("Reports extensiveness"))
+  setDefault(reportType, ReportParam.Extended())
+
+  def getReportType: ReportType = $(reportType)
+
+  def setReportType(value: ReportType): this.type = set(reportType, value)
 }
 
 object DOperation {
   type Id = models.Id
   val Id = models.Id
+
+  object ReportParam {
+    case class SchemaOnly() extends ReportType {
```

Comment:
rename to Minimal? or Simplified?

Reply:
Minimal reports are generated when there are over 100 rows.

Reply:
Well, the name of this param says "Minimal report". Let's figure out some sensible name and have name of this class match the 'name' field.